spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -455,7 +462,7 @@ public ConfigurableApplicationContext run() {
 
 	}
 
-	@EnableAutoConfiguration
+	@ImportAutoConfiguration({ChannelBindingAutoConfiguration.class, EndpointAutoConfiguration.class})
```

Comment:
@jvalkeal The `@ImportAutoConfiguration` only imports the binding related auto configs which are internal to SCSt; and this is something user wouldn't want to get control over.

> Maybe we could add some hooks to replace this config if user so want/need

We already do that where we use external auto configuration (EmbeddedServlet auto config)